modules/flowable-engine/src/test/java/org/flowable/engine/test/api/task/TaskIdentityLinksTest.java

```diff
@@ -32,6 +35,8 @@
 
 import com.fasterxml.jackson.databind.node.ArrayNode;
 
+import static org.assertj.core.api.Assertions.*;
```

Comment:
Thanks for the PR.  Project formatting standards are not to use package imports but rather enumerate them all.  In addition static imports are at the top of the list of import statements.